src/popover/styles.scss

```diff
@@ -15,7 +15,6 @@
 .root {
   @include styles.styles-reset;
   color: inherit;
-  display: inline;
```

Comment:
Not needed after replacing `div` with `span` above. The new test page confirms this too